### PR TITLE
add CR appropriation transaction

### PR DESCRIPTION
**Pull request by RainFallsSilent**

### CODE REVIEW

File: cr/state/committee.go [hunk 1]
```diff
@@ -9,6 +9,7 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
+	"github.com/elastos/Elastos.ELA/account"
```
Comment (raindust) [inline cr/state/committee.go]: format

File: settings.go [hunk 2]
```diff
@@ -391,6 +391,36 @@ func newSettings() *settings {
 		ConfigPath: "CRCAddress",
 		ParamName:  "CRCAddress"})
 
+	result.Add(&settingItem{
+		Flag:         nil,
```
Comment (raindust) [inline settings.go]: add CRCFoundationFlag here

File: cr/state/committee.go (hunk 1 above)
Comment (RainFallsSilent) [reply cr/state/committee.go]: fixed

File: settings.go (hunk 2 above)
Comment (RainFallsSilent) [reply settings.go]: fixed

**Pull request merged**